.github/workflows/pull_request.yml

```diff
@@ -5,13 +5,13 @@ on:
       - main
   pull_request:
 
+env:
+  FDB_VER: "6.3.24"
```

Comment:
TODO revert back to 6.3.24, see: https://github.com/apple/foundationdb/issues/6872